ods_ci/utils/scripts/rosa/rosa.py

```diff
@@ -21,6 +23,22 @@ def __init__(self, args={}):
         self.rosa_version = args.get("rosa_version")
         self.channel_name = args.get("channel_name")
 
+    def set_rosa_version(self):
+        version_match = re.match(r"(\d+\.\d+)\-latest", self.rosa_version)
+        if version_match is None:
+            log.info(f"Using the rosa version given by user: {self.rosa_version}")
+            return
```

Comment:
Maybe we can include some info message also for the `else` branch? Something like:
```
User provided {self.rosa_version} OpenShift version, trying to determine appropriate latest version for the ROSA cluster creation.
```